io/plugins/eu.esdihumboldt.hale.io.csv.test/src/eu/esdihumboldt/hale/io/csv/CSVInstanceReaderTest.java

```diff
@@ -246,11 +280,47 @@ private Schema readCSVSchema(String sourceLocation, String typeName, String para
 		return schemaReader.getSchema();
 	}
 
+	private InstanceCollection readCSVInstances(String sourceLocation, String typeName, int skipN,
+			Schema sourceSchema, String seperator, String quote, String escape) throws Exception {
+
```

Comment:
I believe that the method body can be replaced with 
```
return readCSVInstances(sourceLocation, typeName, skipN, sourceSchema, seperator, quote,
 				escape, null);
```
This will call your method:
```
private InstanceCollection readCSVInstances(String sourceLocation, String typeName, int skipN,
 			Schema sourceSchema, String seperator, String quote, String escape, String decimal)
```
with decimal parameter as `null`and will avoid code duplication.

Reply:
I understand that I can substitute the two kinds of return. But I don't understand where I can avoid code duplication.